### DISCUSSION TITLE
Welcome to The-Lords-Of-Midnight Discussions!

### DISCUSSION THREAD

**Discussion by IcemarkUK** [ChilliHugger/The-Lords-Of-Midnight#73] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by MarkusRamikin** [comment]
In releases I only see the source. Any chance for a windows installer, or portable zip, please, for us ordinary mortals?

**Reply by fassisrosa** [reply]
You can try these links:

https://thelordsofmidnight.com/blog/the-lords-of-midnight-v2-0/

https://thelordsofmidnight.com/blog/doomdarks-revenge-v2-0/

**Comment by FrozenEmpire** [comment]
Hi Chris,

Can you please check the logic for using The Crown of Imiriel in ddr_character.cpp

If Shareth uses this Crown, what is the intended result - is it that Luxor and all those loyal to the Moonprince get transported to Shareth's location ?

ddr_character::Cmd_Use  seems to drop through to the default case when the Crown of Imiriel is being carried, which has a different result...

Maybe I'm not understanding it 100% though

Thanks
Andrew

**Reply by IcemarkUK** [reply]
Responded here... https://github.com/ChilliHugger/The-Lords-Of-Midnight/discussions/165
